tests/core/tests/test_resources/basic_resources.py

```diff
@@ -0,0 +1,198 @@
+from collections import OrderedDict
+from unittest import mock
+from unittest.mock import patch
+
+import tablib
+from django.test import TestCase
+
+from import_export import fields, resources, results
+from tests.core.models import Author, Book, Category, Profile, WithDefault
+
+
+class MyResource(resources.Resource):
```

Comment:
I suggest take all Resource classes out of test modules and put them in a new module called 'resources`, that are then imported into test modules.  This way we separate tests and helper classes.

Reply:
Just want to clarify if we want to keep the resources module in folder `tests/core` or totally outside of parent `tests` folder.

Reply:
The resources in here should apply to tests only, so I suggest to move to a module in 

```
tests/core/tests
```